BUILDING.md

```diff
@@ -15,9 +16,22 @@
     $ cd <clone>
     $ mkdir build
     $ cd build
-    $ cmake ../src
+    $ cmake ..
     $ cmake --build . -- /m
 
+## Optional Build Parameters
+
+-DGEODE_ROOT=<path to Apache Geode binaries>
+ This will allow you to specify the path to use for the Apache Geode libraries and binaries
+
+On Windows
```

Comment:
This is not relevant to CMake's configure phase like -D options above are. You should distinguish between the two.  Be careful you don't re-invent CMake docs.